torch/utils/__init__.py

```diff
@@ -28,13 +29,17 @@ def swap_tensors(t1, t2):
     At a high level, this will make t1 have the content of t2 while preserving
     its identity.
 
-    This will not work if t1 and t2 have different number of slots.
+    This will not work if t1 and t2 have different slots.
```

Comment:
This change because of [this](https://github.com/python/cpython/blob/e1117cb8864ca337a26388a6186aa85f7cc29a94/Doc/reference/datamodel.rst?plain=1#L2243-L2244)

> :attr:`~instance.__class__` assignment works only if both classes have the
  same *__slots__*.